### DISCUSSION TITLE
Are any CMSs already integrating the Block Protocol?

### DISCUSSION THREAD

**Discussion by fgatti675** [blockprotocol/blockprotocol#311] [Q&A] [answered]
Hi everyone!
First of all, I love the idea of the Block protocol.
It's something I've had in my mind for years and I am really happy that it has bee done in an open way.
I would like to ask you if you know of any CMS who is already integrating this technology (I know Wordpress has planned it)
We are developing a CMS based on Firestore (probably agnostic in the future) and would like to make the Blocks protocol a first class citizen.
Our CMS is called FireCMS https://firecms.co/ and it is based on React
Any tips on how to proceed or even collaborators would be awesome :)

**Comment by jtewright** [comment, marked as answer]
Hi @fgatti675! I work on Product at HASH/Block Protocol. We're glad you love the project :)

In addition to [WordPress](https://twitter.com/nonparibus/status/1529210576865357824), we're talking to a number of potential partners, including some other CMSs. FireCMS looks awesome and I can definitely see how it could leverage blocks. I will send you an email to chat about collaborating in more detail.

Joseph
